docs/players/creating-bots/agent-builder.mdx

```diff
@@ -201,27 +201,28 @@ You can close validation results at any time and reopen them by clicking the cli
 
 <SavingYourTree />
 
-## Editing an AI's source code
-
-Whenever you save your tree, multiple files are pushed to GitHub - a config file named `agentBuilderWorkspace.json` which defines the nodes in your tree,
-and a series of typescript files generated from the config that are used to run your AI in-game.
-
-```
-├── lib
-│   ├── BaseClasses
-│   │   ├── **/*.ts
-│   ├── **/*.ts
-├── agentBuilderWorkspace.json
-├── index.ts
-├── package.json
-├── tsconfig.json
-└── .gitignore
-```
-
-You can modify these files in your favorite IDE for more control over the AI's behaviors and dependencies.
-See the [Writing Your Own Code section](./coding/dev-requirements) for more information.
-
-:::danger
-Changes made to files other than the `agentBuilderWorkspace.json` will be overwritten when you save the tree from the Agent Builder interface.
-If you plan to modify files manually, it's a good idea to create a new GitHub repository to prevent losing your work.
-:::
+[//]: # (TODO [REG-1386] update this section)
```

Comment:
I assume this syntax is commenting this out? Can we just remove it and rely on Git history? Or, if you want to preserve it somewhere a little easier, maybe copy this old code in to REG-1386?

I generally prefer if we can avoid committing commented-out code since it adds extra noise that can quickly get out of date. If we don't address REG-1386 for six months (could happen, who knows how our priorities will shift 😄) then this noise will still be here.

Reply:
Yes, those are markdown comments :-)

Removed